medusa/databases/__init__.py

```diff
@@ -1,20 +1,3 @@
 # coding=utf-8
 
```

Comment:
We need to be adding module docstrings regardless.  We also need to add a license file.